vantage/virtual_tag_config_resource.go

```diff
@@ -53,13 +58,47 @@ func (r VirtualTagConfigResource) Schema(ctx context.Context, req resource.Schem
 				Computed: true,
 				NestedObject: schema.NestedAttributeObject{
 					Attributes: map[string]schema.Attribute{
-						"filter": schema.StringAttribute{
+						"business_metric_token": schema.StringAttribute{
 							Optional:            true,
-							Computed:            true,
+							Description:         "The token of the associated BusinessMetric.",
```

Comment:
There are validators available for enforcing attribute exclusivity in the provider. Something to possibly look at later. If it can't be generated from swagger, I'm less interested in enriching our client in a bespoke way vs letting the API validate.